### DISCUSSION TITLE
Missing SelectAllTextBehavior in Nugat Package? Missing Doc?

### DISCUSSION THREAD

**Discussion by s-beji** [CommunityToolkit/Maui#480] [Q&A] [answered]
I wanna use that component but i dont want to manuly link the github source project with mine i rather use the nugat package like i already do but now i wanted to implement it and saw it isn't included in the libary. I checked my version its 1.0.0

![image](https://user-images.githubusercontent.com/59297046/176942850-957da214-57a9-454b-b447-3e63b8e7c327.png)

**Comment by bijington** [comment, marked as answer]
@DeepWorksStudios This feature hasn't been released yet. It will be coming soon but I don't have a date to give you I'm afraid

**Comment by CliffAgius** [comment]
Closed as answered
